### PR TITLE
[jeelink] Add suport for sensors that are directly connected to a LGW

### PR DESCRIPTION
**Pull request by vbier**
This adds support for sensors that are directly connected to Lacrosse Gateways.

See https://community.openhab.org/t/new-jeelink-openhab2-binding/15018/370 ff.

Signed-off-by: Volker Bier <volker.bier@web.de>

### CODE REVIEW

File: bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwReadingConverter.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import org.openhab.binding.jeelink.internal.JeeLinkReadingConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Converter for converting a line read from a LGW to a LgwReading.
+ *
+ * @author Volker Bier - Initial contribution
+ */
+public class LgwReadingConverter implements JeeLinkReadingConverter<LgwReading> {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwReadingConverter.java]: Please add `@NonNullByDefault`

File: bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwSensorDefinition.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import org.openhab.binding.jeelink.internal.JeeLinkReadingConverter;
+import org.openhab.binding.jeelink.internal.JeeLinkSensorHandler;
+import org.openhab.binding.jeelink.internal.SensorDefinition;
+
+/**
+ * Sensor definition of a sensor directly connected to a LGW.
+ *
+ * @author Volker Bier - Initial contribution
+ */
+public class LgwSensorDefinition extends SensorDefinition<LgwReading> {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwSensorDefinition.java]: see above

File: bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwSensorHandler.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+import org.openhab.binding.jeelink.internal.ReadingPublisher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handler for a LGW Sensor thing.
+ *
+ * @author Volker Bier - Initial contribution
+ */
+public class LgwSensorHandler extends JeeLinkSensorHandler<LgwReading> {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwSensorHandler.java]: see above

File: bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwReadingConverter.java (hunk 1 above)
Comment (vbier) [reply bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwReadingConverter.java]: LgwReadingConverter.java:[35,1239]: Illegal redefinition of parameter inputLine, inherited method from JeeLinkReadingConverter<LgwReading> does not constrain this parameter

File: bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwSensorDefinition.java (hunk 2 above)
Comment (vbier) [reply bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwSensorDefinition.java]: LgwSensorDefinition.java:[45,1443]: Illegal redefinition of parameter thing, inherited method from SensorDefinition<LgwReading> does not constrain this parameter

File: bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwSensorHandler.java (hunk 3 above)
Comment (vbier) [reply bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwSensorHandler.java]: This is the only place where the compiler does not complain about the annotation.

**Pull request merged**